pom.xml

```diff
@@ -125,7 +129,7 @@
     <displaytag.version>1.2</displaytag.version>
     <dom4j.version>1.6.1</dom4j.version>
     <dwr.version>3.0.RC2</dwr.version>
-    <eclipselink.version>2.5.2</eclipselink.version>
+    <eclipselink.version>2.6.0-M3</eclipselink.version>
```

Comment:
As discussed during a standup, it's OK to upgrade eclipselink to a milestone version. Please add a jira to upgrade it to the release version before we release 2.6.0.